strings/names/death_commando.txt

```diff
@@ -61,7 +61,7 @@ Splint Chesthair
 Stabby McGee
 Stump Beefgnaw
 Stump Chunkman
-THAT DAMN FAGGOT TRAITOR GEORGE MELONS
+THAT DAMN meatball TRAITOR GEORGE MELONS

```

Comment:
Same here

Reply:
This is intentional. The sudden all lowercase on a singular word makes it obvious it once was something else. Perhaps if they look at the blame, they can find what it was, making it not only funny in game, but out of game as well.